librosa/core/audio.py

```diff
@@ -72,6 +72,9 @@ def load(
         If the codec is supported by `soundfile`, then `path` can also be
         an open file descriptor (int).
 
+        If the codec is not supported by `soundfile` (for example, mp3),
```

Comment:
On the contrary, if the codec is not supported by `soundfile` (for example, MP3),
then `path` must be a file path (string or pathlib.Path).